api/coreservice.go

```diff
@@ -1635,3 +1637,10 @@ func (core *coreService) SimulateExecution(ctx context.Context, addr address.Add
 	exec.SetGasLimit(core.bc.Genesis().BlockGasLimit)
 	return core.sf.SimulateExecution(ctx, addr, exec, core.dao.GetBlockHash)
 }
+
+// SyningProgress returns the syncing status of node
+func (core *coreService) SyningProgress() (uint64, uint64, uint64) {
+	return core.dao.StartingHeight(),
+		core.bc.TipHeight(),
+		core.bs.TargetHeight()
```

Comment:
should return `"result": false` if the chain is not syncing (reaches highest block)